react/features/base/color-scheme/ColorSchemeRegistry.ts

```diff
@@ -34,13 +33,13 @@ class ColorSchemeRegistry {
     /**
      * Retrieves the color-scheme applied style definition of a component.
      *
-     * @param {Object | Function} stateful - An object or function that can be
+     * @param {IStateful} stateful - An object or function that can be
      * resolved to Redux state using the {@code toState} function.
      * @param {string} componentName - The name of the component whose style we
      * want to retrieve.
      * @returns {StyleType}
      */
-    get(stateful: Object | Function, componentName: string): StyleType {
+    get(stateful: IStateful, componentName: string): StyleType {
```

Comment:
This whole thing is legacy and will go away. I think it's best not to "modernize" it.